<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackModelModule</class>
 <widget class="QMainWindow" name="TrackModelModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Track Model</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="ModuleSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1256</width>
      <height>621</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="TrackLineColorValue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-10</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="maxCount">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>Select a Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green Line</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="TrackModelBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>1211</width>
       <height>631</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Track Model</string>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>21</y>
        <width>1221</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="SetEnvironmentTemperatureInputBox">
     <property name="geometry">
      <rect>
       <x>801</x>
       <y>21</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="value">
      <number>68</number>
     </property>
    </widget>
    <widget class="QLabel" name="EnvironmentTemperatureLabel">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Environment Temperature (degrees Fahrenheit)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadTrackModelButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Track Model</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="FailureButtons">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="TrackCircuitFailureToggleButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Track Circuit Failure</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="BrokenRailToggleButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Broken Rail</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="PowerFailureToggleButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Power Failure</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>TrackCircuitFailureToggleButton</zorder>
     <zorder>PowerFailureToggleButton</zorder>
     <zorder>BrokenRailToggleButton</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="MenuSection">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>1256</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 175, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="ModuleSelectionMenu">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 215)</string>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="maxCount">
      <number>12</number>
     </property>
    </widget>
    <widget class="QPushButton" name="SystemInfoButton">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>20</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 215);
color: black;</string>
     </property>
     <property name="text">
      <string>System Info</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SystemSpeedValue">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
     </property>
     <property name="suffix">
      <string> x</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QTimeEdit" name="SystemClockValue">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>20</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>13</hour>
       <minute>56</minute>
       <second>59</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>13</hour>
       <minute>56</minute>
       <second>59</second>
      </time>
     </property>
     <property name="minimumTime">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>h:mm</string>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="SystemSpeedLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>System Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SystemBlockLabel">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>System Clock:</string>
     </property>
    </widget>
    <zorder>SystemInfoButton</zorder>
    <zorder>SystemSpeedValue</zorder>
    <zorder>SystemClockValue</zorder>
    <zorder>ModuleSelectionMenu</zorder>
    <zorder>SystemSpeedLabel</zorder>
    <zorder>SystemBlockLabel</zorder>
   </widget>
   <widget class="QLabel" name="FailureModesBoxLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Failure Modes</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../../private/var/folders/ng/01ftyqy97f76p5nf0wp0m97c0000gn/T/com.apple.Photos.NSItemProvider/version=1&amp;uuid=D09791C0-EEBA-4D5E-BF34-686A3C7DDA6D&amp;mode=compatible&amp;noloc=0.jpeg/IMG_2625.jpeg"/>
 </resources>
 <connections/>
</ui>
